projects/open-mpi.org/package.yml

```diff
@@ -0,0 +1,73 @@
+distributable:
+  url: https://download.open-mpi.org/release/open-mpi/v{{version.marketing}}/openmpi-{{version}}.tar.bz2
+  strip-components: 1
+
+versions:
+  github: open-mpi/ompi/tags
+
+dependencies:
+  open-mpi.org/hwloc: '*'
+  libevent.org: '*'
+  linux:
+    gnu.org/gcc: '*'
```

Comment:
Is this really a run-time and not a build-time dep? And doesn't darwin need a compiler as well?

Reply:
<img width="761" alt="image" src="https://github.com/teaxyz/pantry/assets/51118083/fba3f64c-982d-4f11-a48c-447f50a7cfee">
the homebrew formula states that gcc is needed for gfortran
hmm i must have excluded it for darwin during one of my attempts to assemble it and forgot to return it 😄

Reply:
fun. weird that it'd be runtime.

Reply:
If I return gcc for darwin I get:
```
configure:6591: checking whether the C compiler works
configure:6613: gcc -w   -Wl,-rpath,/Users/andrejrabcenko/.tea conftest.c  >&5
ld: library not found for -lSystem
collect2: error: ld returned 1 exit status
```